src/main/resources/db/migration/V8_1__add_communication_table.sql

```diff
@@ -0,0 +1,47 @@
+create table communication
+(
+    viewed           bit,
+    sent             datetime(6),
+    communication_id varchar(255) not null,
+    email            varchar(255),
+    errand_number    varchar(255),
+    external_case_id varchar(255),
+    message_body     varchar(255),
+    mobile_number    varchar(255),
+    subject          varchar(255),
+    direction        enum ('INBOUND','OUTBOUND'),
+    message_type     enum ('SMS','EMAIL'),
+    primary key (communication_id)
+) engine = InnoDB;
+
+create table communication_attachment
+(
+    communication_attachment_data_id integer      not null,
+    attachment_id                    varchar(255) not null,
+    communication_id                 varchar(255),
+    content_type                     varchar(255),
+    name                             varchar(255),
+    primary key (attachment_id)
+) engine = InnoDB;
+
+create table communication_attachment_data
+(
+    id   integer not null auto_increment,
+    file longblob,
+    primary key (id)
+) engine = InnoDB;
+
+
+create index idx_errand_number
+    on communication (errand_number);
+
+alter table if exists communication
+    add constraint uq_errand_number unique (errand_number);
+
+alter table if exists communication_attachment
+    add constraint UK_k3pya9ygw4dkeaqoe4sv733jx unique (communication_attachment_data_id);
```

Comment:
A more user friendly name for this one perhaps :)

Reply:
I think that this is due to a bug in spring-data? I have seen it before.

Reply:
yeah there is some buggy behaviour with this. But could and should probably name it properly in the flyway script anyway :)